apps/server/src/modules/health/domain/healthcheck.do.ts

```diff
@@ -0,0 +1,10 @@
+export class Healthcheck {
```

Comment:
This class should be named `HealthCheck` also to be consistent.
```suggestion
export class HealthCheck {
```

Reply:
Done in https://github.com/hpi-schul-cloud/schulcloud-server/pull/4191/commits/9f7cb45820e06668af950204e05f82983c112e54 .